docs/omero-web-release-process.rst

```diff
@@ -0,0 +1,59 @@
+OMERO.web release
+=================
+
+.. _OMERO Web: https://github.com/ome/omero-web/
+.. _image.sc: https://forum.image.sc/
+
+This document describes the release process for `OMERO Web`_.
+The release process uses GitHub actions, make sure that the actions are active before pushing any tag.
+
+
+Release process
+^^^^^^^^^^^^^^^
+
+Source code release
+-------------------
+
+.. seealso::
+
+    :doc:`python-development`
+       Release process of Python components
```

Comment:
Alternatively this could (also ?)link to https://github.com/ome/omero-web#release-process.

Reply:
done